docs/0_overview.md

```diff
@@ -0,0 +1,44 @@
+# Overview
+
+Crossplane is an open source control plane that allows you to manage
+infrastructure the Kubernetes way. It provides the following features:
+
+- Deployment and management of cloud provider managed services using the
+  Kubernetes API.
+- Management and scheduling of configuration data across multiple Kubernetes
+  clusters.
+- Separation of concern between [infrastructure owners](), [application
+  owners](), and [developers]().
+- Infrastructure agnostic packaging of applications and their dependencies.
+
+Crossplane does not:
+
+- Require that you run your workloads on Kubernetes.
+- Manage the data plane across Kubernetes clusters.
+- Manage or provision non-hosted Kubernetes clusters.
+
+Crossplane can be [installed]() into any Kubernetes cluster, and is compatible
+with any Kubernetes-native project. It manages external services by installing
+[Custom Resource Definitions]() (CRDs) and [reconciling]() instances of those
+Custom Resources. Crossplane is built to be extensible, meaning that anyone can
+add functionality for an new or existing cloud provider.
+
+Crossplane is comprised of four main components:
+
+1. **Core Crossplane**: the set of Kubernetes CRDs and controllers that manage
+   installation of `providers`, `stacks`, and `applications`, as well as the
+   scheduling of configuration data to remote Kubernetes clusters.
+2. **Providers**: the set of Kubernetes CRDs and controllers that provision and
+   manage services on cloud providers. A cloud provider is any service that
+   exposes infrastructure via an API.
+    - Examples: [Google Cloud Platform](), [Amazon Web Services](), [Azure](),
+      [Alibaba](), [Github]()
+3. **Stacks**: a bundled set of Custom Resource instances that together
```

Comment:
```suggestion
3. **Stacks**: a bundled set of Custom Resource instances and classes that together
```

Reply:
Classes are themselves a custom resource. I revised this sentence though, trying to talk kubernetes language in the overview then get into Crossplane language farther down the line.